grails-app/services/au/org/ala/downloads/LoggerService.groovy

```diff
@@ -97,6 +98,11 @@ class LoggerService {
         final port = grailsApplication.config.app.logger.port ?: '80'
         final path = grailsApplication.config.app.logger.path ?: '/service/logger/'
         final reasons = grailsApplication.config.app.logger.reasons ?: 'reasons'
-        httpWebService.getJson("${server}:${port}${path}${reasons}")
+        def resp = webService.get("${server}:${port}${path}${reasons}")
+        if (resp.error) {
```

Comment:
resp.statusCode > 299 again (I'd like to augment the ws plugin to return a response object that can calculate a simple response.error property though (or replace all uses of this plugin with Retrofit!))